app/src/controllers/log/sagas.js

```diff
@@ -73,6 +76,20 @@ function* fetchActivity() {
   yield take(createFetchPredicate(ACTIVITY_NAMESPACE));
 }
 
+function* fetchMoreHistoryItems() {
```

Comment:
Seems that this saga is too similar with `fetchHistoryItems` [saga](https://github.com/reportportal/service-ui/pull/2601/files#diff-f331d34263a8765d03a750f31a61d1d83cd24a2109cb6164d648eb1e72f62c52L111).
Can we reuse `fetchHistoryItems` saga to handle action with additional payload parameter (f.e. `loadMore: true`)?